src/dispatch/alembic/versions/c6db1c7655db_.py

```diff
@@ -0,0 +1,32 @@
+"""Adds evergreen data for documents.
```

Comment:
This file should not be part of this PR. It's already in #660

Reply:
Yeah, I backported it so that I could generate the new fields I needed on the database. It should merge cleanly?

Reply:
I guess so.